share/resources/gflags.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
That's great.  One question,  how to distinguish different module,  storage/graph/meta  has different gflags.

Reply:
For now it seems we can apply theres to all storage/graph/meta, if we do need to distinguish modules of a gflag, I will add another layer to indicate modules.

Reply:
Yes, we need it.   Maybe we will start graph/meta/storage on the same server